core/src/org/pentaho/di/core/jdbc/ThinDriver.java

```diff
@@ -67,6 +68,11 @@ public Connection connect( String url, Properties properties ) throws SQLExcepti
     String username = properties.getProperty( "user" );
     String password = properties.getProperty( "password" );
 
+    Logger.getLogger( getClass().getName() ).log( Level.WARNING,
+      getClass().getName() + " has been deprecated and is scheduled for removal in 6.1. "
+        + "Please use org.pentaho.di.trans.dataservice.jdbc.ThinDriver instead."
```

Comment:
Yes ... but please add a @deprecated comments in this class using proper notation (http://wiki.pentaho.com/display/PEOpen/Deprecation+Guidelines)